web/src/polyfills.ts

```diff
@@ -41,8 +41,8 @@
 
 
 /** Evergreen browsers require these. **/
-import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
+// import 'core-js/es6/reflect';
```

Comment:
Can these be removed?

Reply:
It would be hypocritical otherwise 😄 
I'll remove it now